scripts/coredump/coredump_gdbserver.py

```diff
@@ -43,6 +43,8 @@ def parse_args():
 
     parser.add_argument("elffile", help="Zephyr ELF binary")
     parser.add_argument("logfile", help="Coredump binary log file")
+    parser.add_argument("--additionalelfs", help="Additional ELF binaries in a comma separated list")
```

Comment:
I think you can use `action="extend"` so that the argument can be put into a list, and no need to be comma separated. With that, you can use the comma to specify the text address so the address always goes together with the elf file, instead of having to match two different argument lists (which is prone to error).

Reply:
Thanks, updated